codegenerator/cli/src/hbs_templating/codegen_templates.rs

```diff
@@ -1182,13 +1182,13 @@ impl FieldSelection {
     }
 
     fn aggregated_selection(cfg: &system_config::SystemConfig) -> Self {
-        let mut transaction_fields: HashSet<_> = cfg
+        let mut transaction_fields: BTreeSet<_> = cfg
```

Comment:
This had non deterministic ordering, just changed to a BTreeSet to keep ordering consistent.